src/components/appmodalcartselect.main.jsx

```diff
@@ -71,48 +62,69 @@ class AppModalCartSelectMain extends React.Component {
   }
 
   fetchOrganizationData() {
-    cortexFetch(`/admin_eam?zoom=${zoomArray.join()}`, {
-      headers: {
-        'Content-Type': 'application/json',
-        Authorization: localStorage.getItem(`${Config.cortexApi.scope}_oAuthTokenEam`),
-      },
-    })
-      .then(res => res.json())
-      .then((res) => {
-        const orgEamData = res._authorizationcontexts[0]._element.find(element => element.name === Config.cortexApi.scope.toUpperCase());
-        this.setState({
-          orgEamData,
-        });
-      })
-      .catch((error) => {
-        // eslint-disable-next-line no-console
-        console.error(error.message);
+    login()
```

Comment:
The `login` will return a 404 at this state and never makes it into the `admin_eam` call.
![image](https://user-images.githubusercontent.com/39800563/51788637-4fb5f000-214e-11e9-847e-055c917f11a7.png)